msteams-platform/tabs/how-to/using-fluid-msteam.md

```diff
@@ -0,0 +1,325 @@
+2. Create a Teams tab application by running the below command and following [the Create a channel tab documentation](create-channel-group-tab.md#create-a-custom-channel-or-group-tab-with-nodejs).
+
+    ```cmd
+    yo teams
+    ```
+
+3. After creating the project, navigate to it with the command `cd <your project name>`.
+4. The project will use three Fluid libraries:
+
+    |Library |Description |
+    |---|---|
+    | `fluid-framework`    |Contains the IFluidContainer and other [distributed data structures](https://fluidframework.com/docs/build/dds/) that synchronize data across clients.|
+    | `@fluidframework/azure-client`   |Defines the starting schema for the [Fluid container](https://fluidframework.com/docs/build/containers/).|
+    | `@fluidframework/test-client-utils` |Defines the `InsecureTokenProvider` needed to create the connection to a Fluid service.|
+
+    Run the following command to install the libraries.
+
+    ```cmd
+    npm install @fluidframework/azure-client fluid-framework @fluidframework/test-client-utils
+    ```
+
+## Code the project
+
+1. Open the file `/src/client/<your tab name>` in your code editor. Create a new file named `Util.ts`. Then add the following import statements:
+
+    ```ts
+    //`Util.ts
+
+    import { IFluidContainer } from "fluid-framework";
+    import { AzureClient, AzureClientProps } from "@fluidframework/azure-client";
+    import { InsecureTokenProvider } from "@fluidframework/test-client-utils";
+    ```
+
+### Defining Fluid functions and parameters
+
+1. This app is intended to be used in the context of Microsoft Teams, having all Fluid-related imports, initialization, and functions in one place makes it easier to use in the future. Add the following code below the import statements. The comments will define all the functions and constants needed to interact with the Fluid service and container.
+
+```ts
+// TODO 1: Define the parameter key(s).
+// TODO 2: Define container schema.
+// TODO 3: Define connectionConfig (AzureClientProps).
+// TODO 4: Create Azure client.
+// TODO 5: Define create container function.
+// TODO 6: Define get container function.
+```
+
+1. Replace `TODO 1:` with the following code. The constant is being exported because it will be appended to the `contentUrl` in the Microsoft Teams settings and later for parsing the container ID in the content page. It's a common pattern to store important query parameter keys as constants, rather than typing the raw string each time.
+
+```ts
+export const containerIdQueryParamKey = "containerId";
+```
+
+1. Before the client can create any containers, it needs a `containerSchema` that will define the shared objects used in this application. This example uses a SharedMap as the `initialObjects`, but any shared object can be used. Note also that `map` is the ID of the `SharedMap` object and it must be unique within the container as with any other DDSes. Replace `TODO: 2` with the following code.
```

Comment:
1. Before the client can create any containers, it needs a `containerSchema` that define the shared objects used in this application. This example uses a SharedMap as the `initialObjects`, but any shared object can be used. 

> [!Note] 
The `map` is the ID of the `SharedMap` object and it must be unique within the container as with any other DDSes.